examples/jsm/loaders/GLTFLoader.d.ts

```diff
@@ -29,3 +31,19 @@ export class GLTFLoader extends Loader {
 	parse( data: ArrayBuffer | string, path: string, onLoad: ( gltf: GLTF ) => void, onError?: ( event: ErrorEvent ) => void ) : void;
 
 }
+
+export class GLTFParser {
+
+	json: any;
+	extensions: any;
+	options: {
+		path: string,
+		corssOrigin: string,
+		manager: LoadingManager
+	};
+
+	parse: ( onLoad: ( result: GLTF ) => void, onError: ( reason: any ) => void ) => void;
+	getDependency: (type: string, index: number) => Promise<any>;
+	getDependencies: (type: string) => Promise<any[]>;
```

Comment:
I picked members that I assumed that is intended to be exposed. might be more or less

Reply:
I don't think users should need to call `parse()` directly; at least in the current API parsing will already be done by the time they receive the GLTFParser. The other two methods are useful, though.

Reply:
sure, I'm going to drop the `parse`